### PR TITLE
Parser for logfile var log up2date 3.x

### PR DESCRIPTION
**Pull request by TripleJQK**

### CODE REVIEW

File: insights/specs.py [hunk 1]
```diff
@@ -355,6 +355,7 @@ class Specs(SpecSet):
     udev_persistent_net_rules = RegistryPoint(alias="udev-persistent-net.rules")
     uname = RegistryPoint()
     up2date = RegistryPoint()
+    up2date_log = RegistryPoint(alias="up2date_log")
```
Comment (xiangce) [inline insights/specs.py]: `alias` is used to keep compatible with the name in 1.x which contains invalid characters (`.` or `-`), For the newly added specs, it's good to use the same name in both 1.x and 3.x

**Comment by TripleJQK** [comment]
changed as your requested:
delele alias, use up2date_log as same as 1.x
edit the timestamp

File: insights/parsers/up2date_log.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from .. import LogFileOutput, parser
```
Comment (xiangce) [inline insights/parsers/up2date_log.py]: Please add the corresponding docstring file header. Refer to the `insights/parsers/httpd_V.py` (rebase to the latest master please)

Comment (bfahr) [reply insights/parsers/up2date_log.py]: Also please add a `.rst` file to include this module in the documentation as described here: http://insights-core.readthedocs.io/en/latest/docs_guidelines.html#documentation-guidelines

**Comment by TripleJQK** [comment]
@xiangce modified as your request, please review when free

File: insights/parsers/up2date_log.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+"""
+up2date Logs -  Files ``/var/log/up2date``
+==========================================
+
+Modules for parsing the content of log file ``/var/log/up2date`` in sosreport archives of RHEL.
+
+.. note::
```
Comment (xiangce) [inline insights/parsers/up2date_log.py]: This `Note` is duplicated with the one inner the class,  better remove this one.

**Pull request merged**